src/librustc/diagnostics.rs

```diff
@@ -279,6 +279,41 @@ println!("{}", Y);
 ```
 "##,
 
+E0019: r##"
+A function call isn't allowed in the const's initialization expression
+because the expression's value must be known at compile-time. Bad example:
```

Comment:
The phrase "Bad example" is often used when one is saying that the example itself is bad -- i.e. when the example does not actually serve as a demonstration of the topic at hand.

I would instead write: "Example of erroneous code:" or "Example of non-compiling code:"